extensions/blocks/podcast-player/templates/playlist-track.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Podcast Title template.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Podcast_Player;
+
+// phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$class = 'jetpack-podcast-player__track';
+$style = '';
+
+if ( $is_active ) {
+	$class .= " is-active {$primary_colors['class']}";
+
+	$style = $primary_colors['style'];
+} else {
+	$class .= " is-active {$secondary_colors['class']}";
+
+	$style = $secondary_colors['style'];
+}
```

Comment:
Should the `is-active` class only be added when `$is_active` is true?

This could be simplified to:

```suggestion
$class = 'jetpack-podcast-player__track is-active ' . $secondary_colors['class'];
$style = $secondary_colors['style'];

if ( $is_active ) {
	$class = 'jetpack-podcast-player__track is-active ' . $primary_colors['class'];
	$style = $primary_colors['style'];
} 
```

Reply:
nop, `is-active` need to be applied once, in the first track :-( . Updating...

Reply:
gong to update your commit but keeping in mind that `is-active` shouldn't be applied to all tracks in the playlist